### PR TITLE
Feature/sshd customization

### PR DESCRIPTION
**Pull request by MimiXav**

### CODE REVIEW

**Review by jeffdegott — APPROVED** [review]

**Comment by jimklimov** [comment]
I am not sure this would work: on one hand, you probably meant /etc/pam.d/sshd and on another - files are likely to exist in advance so `ln` would not replace them. It can replace symlinks with `-f` flag though.

**Comment by jimklimov** [comment]
Also, in the age of chroots, containers and such, prefer to forget about making absolute symlinks - to avoid surprises when looking into the OS image from its hoster env, into backups, etc. `ln -srf` is your friend here.

**Comment by jimklimov** [comment]
Also just in case: for our product this change fixes up the R/O root in the OS image. If someone changed these file in a live deployed system (not sure if these two do get configured in practice),  then copies in overlayfs would be applied after upgrade regardless of squashfs changes. If this is not desired, you may want ipc-meta-setup scriptlets instead, to patch an existing overlaid-or-not filesystem for a predictable result.

File: obs/preinstallimage-bios.sh.in [hunk 1]
```diff
@@ -519,6 +519,10 @@ case "$IMGTYPE" in
     *devel*) ;;
     *) sed -i "s|\\(.*pam_cracklib.so\\).*|    password\tinclude\t\tfty-password|" /etc/pam.d/common-password ;;
 esac
+# If sshd template exist, do a symlink on it
+if [ -f /etc/pam.d/template-sshd ]; then
+  ln -s /etc/template-sshd /etc/sshd
```
Comment (perrettecl) [inline obs/preinstallimage-bios.sh.in:524]: use ln -srf

**Review by perrettecl — CHANGES_REQUESTED** [review]

File: obs/preinstallimage-bios.sh.in (hunk 1 above)
Comment (MimiXav) [reply obs/preinstallimage-bios.sh.in]: Done

**Review by perrettecl — APPROVED** [review]

**Comment by jimklimov** [comment]
in init-os-accounts this still changes the parent system (e.g. developer workstation) and not one in ALTROOT (e.g. their container).

Also if you do need the custom nsswitch config in the product OS image, the preinstallimage script by itself currently does not make it happen.

**Pull request merged**